package.json

```diff
@@ -153,5 +153,33 @@
       "path": "./dist/grommet.min.js",
       "maxSize": "110 kB"
     }
+  ],
+  "keywords": [
+    "grommet",
+    "grommet.io",
+    "component",
+    "library",
+    "design system",
+    "framework",
+    "react",
+    "theming",
+    "templates",
+    "sticker sheets",
+    "design patterns",
+    "app templates",
+    "atomic design",
+    "accessibility",
+    "WCAG 2.1",
+    "keyboard navigation",
+    "screen reader tags",
+    "color",
+    "type",
+    "layout",
+    "typgraphy",
```

Comment:
typography
I've no idea why I read it, but now that I did let's fix it. On a funny note, there is a typo while you tried to write typo in typography.